src/smt/set_defaults.cpp

```diff
@@ -1021,6 +1021,11 @@ bool SetDefaults::incompatibleWithIncremental(const LogicInfo& logic,
     reason << "solveIntAsBV";
     return true;
   }
+  if (opts.smt.computePartitions > 0)
```

Comment:
It seems both `0` and `1` are used to essentially disable the whole partitioning stuff. I guess we should use `> 1` here as well?